django_admin_shell/views.py

```diff
@@ -250,4 +252,5 @@ def get_context_data(self, **kwargs):
         ctx['python_version'] = get_py_version()
         ctx['django_version'] = get_dj_version()
         ctx['auto_import'] = str(self.runner.importer)
+        ctx['saved_snippets'] = SavedSnippets.objects.all()
```

Comment:
Can you add some unit tests to check that saved snipets are in `response.context`? 
I think you can extend `test_view.py` file and add there a test `test_saved_snipets`

Reply:
```bash
$ python manage.py test --keepdb django_admin_shell.tests.test_view.ShellViewTest.test_saved_snippets
Using existing test database for alias 'default'...
System check identified no issues (0 silenced).
.
----------------------------------------------------------------------
Ran 1 test in 0.262s

OK
Preserving test database for alias 'default'...
```